src/interfaces/IERC6538Registry.sol

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+/// @dev Interface for calling the `ERC6538Registry` contract to map accounts to their stealth
+/// meta-address. See [ERC-6538](https://eips.ethereum.org/EIPS/eip-6538) to learn more.
+interface IERC6538Registry {
+  /// @dev Emitted when a registrant updates their stealth meta-address.
+  /// @param registrant The account that registered the stealth meta-address.
+  /// @param schemeId The applied stealth address scheme (such as secp25k1).
+  /// @param stealthMetaAddress The stealth meta-address.
+  /// [ERC-5564](https://eips.ethereum.org/EIPS/eip-5564) bases the format for stealth
+  /// meta-addresses on [ERC-3770](https://eips.ethereum.org/EIPS/eip-3770) and specifies them as:
+  ///   st:<shortName>:0x<spendingPubKey>:<viewingPubKey>
+  /// The chain (`shortName`) is implicit based on the chain the `ERC6538Registry` is deployed on,
+  /// therefore this `stealthMetaAddress` is just the `spendingPubKey` and `viewingPubKey`
+  /// concatenated.
+  event StealthMetaAddressSet(
+    bytes indexed registrant, uint256 indexed schemeId, bytes stealthMetaAddress
+  );
+
+  /// @notice Sets the caller's stealth meta-address for the given scheme ID.
+  /// @param schemeId The applied stealth address scheme (such as secp25k1).
```

Comment:
nit: should say something like "The identifier for the applied stealth address scheme, as defined in EIP5564, such as `1` for secp25k1"

Reply:
Now using the same updated docstring everywhere a6e9e3f